src/steps/active-directory/__tests__/converters.test.ts

```diff
@@ -20,21 +20,35 @@ import exampleOrganization from './fixtures/exampleOrganization';
 
 describe('createAccountEntityWithOrganization', () => {
   test('properties transferred', () => {
+    const exampleExtraData = {
+      mobileDeviceManagementAuthority: 'mobileDeviceManagementAuthority',
+      subscriptionState: 'subscriptionState',
+      intuneAccountID: 'intuneAccountID',
+    };
     const accountEntity = createAccountEntityWithOrganization(
       exampleIntegrationInstance,
       exampleOrganization,
+      exampleExtraData,
     );
     expect(accountEntity).toEqual({
       _class: ['Account'],
       _key: 'microsoft_365_account-the-instance-id',
       _type: 'microsoft_365_account',
-      _rawData: [{ name: 'default', rawData: exampleOrganization }],
+      _rawData: [
+        {
+          name: 'default',
+          rawData: { ...exampleOrganization, ...exampleExtraData },
+        },
+      ],
       name: 'Default Directory',
       id: '1111111-1111-1111-1111-111111111111',
       displayName: 'instance.config.name configured by customer',
       defaultDomain: 'verifiedDomain.onmicrosoft.com',
       organizationName: 'Default Directory',
       verifiedDomains: ['verifiedDomain.onmicrosoft.com'],
+      mobileDeviceManagementAuthority: 'mobileDeviceManagementAuthority',
```

Comment:
What do you think of:
```suggestion
      intuneMdmAuthority: 'mobileDeviceManagementAuthority',
```

Reply:
This doesn't have to be `Intune`. I'll add the options to the left but it can be Office 365 basic MDM, Hybrid on-prem Windows MDM, or Intune

Reply:
I think the name makes sense in the context

Reply:
Ahh, I see. Thanks for clarifying!